rules/msk_topic_config_comments.md

```diff
@@ -6,8 +6,8 @@ Topic configurations expressed in milliseconds must have comments explaining the
 The comments can be placed after the property definition on the same line or on the line before the definition.
 
 For computing the human-readable values it considers the following:
-- 1 month has 30 days
-- 1 year has 365 days
+- 1 month has `2629800000` millis which is 30.4375 days
+- 1 year has `31556952000` millis which is 365.2425 days
```

Comment:
maybe it would be also worth to include this information as a comment in msk_topic_config.go?